bin/hokusai

```diff
@@ -118,5 +132,27 @@ def deploy(context, tag):
   """
   sys.exit(hokusai.deploy(context, tag))
 
+@cli.command()
+@click.argument('context', type=click.STRING)
+@click.option('--shell', type=click.STRING, default='bash', help='The shell to run (defaults to bash)')
+@click.option('--tag', type=click.STRING, help='The image tag to run (defaults to the value of context)')
+@click.option('--env', type=click.STRING, multiple=True, help='Environment variables in the form of "KEY=VALUE"')
+def console(context, shell, tag, env):
+  """
+  Launch a new container and attach a shell in the given context
```

Comment:
I hadn't realised these comments are automatically pulled into the cli help output by click :+1: